integration-tests/src/test/java/oracle/kubernetes/operator/ITStickySession.java

```diff
@@ -64,6 +64,7 @@ public static void staticPrepare() throws Exception {
         Map<String, Object> domainMap = TestUtils.loadYaml(DOMAINONPV_WLST_YAML);
         // Treafik doesn't work due to the bug 28050300. Use Voyager instead
         domainMap.put("LB_TYPE", "VOYAGER");
```

Comment:
@hzhao-github you should also use "loadBalancer" attribute, not LB_TYPE in domain map.

Reply:
@hzhao-github I can take care of ITStickySession

Reply:
@hzhao-github @vanajamukkara I will also change Domain.java line 1352's domain map